code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -42,6 +42,19 @@
 	caliber = "trod"
 	projectile_type = /obj/item/projectile/bullet/trod
 
+
+/obj/item/ammo_casing/caseless445m
+	desc = "Caseless ammo fragments."

```

Comment:
If they are caseless they should not be leaving an ammo casing. There is a defined functionality specifically for caseless ammunition.